### DISCUSSION TITLE
How to edit Chakra-ui default toast icons?

### DISCUSSION THREAD

**Discussion by khj0426** [chakra-ui/chakra-ui#8981] [Q&A] [answered]
how can i change chakra-ui default toast icon>

**Comment by segunadebayo** [comment]
In v3, you can edit the `components/ui/toaster` file and modify the icon as needed

**Comment by segunadebayo** [comment, marked as answer]
Pass a custom `icon` on `toaster.create`, or replace `Toaster.Indicator` in the toaster snippet.

```ts
toaster.create({ title: "Saved", icon: <MyIcon /> })
```
